task-sdk/src/airflow/sdk/definitions/deadline.py

```diff
@@ -21,15 +21,20 @@
 from abc import ABC
 from collections.abc import Callable
 from datetime import datetime, timedelta
-from typing import Any, cast
+from typing import TYPE_CHECKING, Any, cast
 
 from airflow.models.deadline import DeadlineReferenceType, ReferenceModels
 from airflow.sdk.module_loading import import_string, is_valid_dotpath
 from airflow.serialization.enums import DagAttributeTypes as DAT, Encoding
 from airflow.serialization.serde import deserialize, serialize
 
+if TYPE_CHECKING:
+    from typing import TypeAlias
+
 logger = logging.getLogger(__name__)
 
+DeadlineReferenceTuple: TypeAlias = tuple[type[ReferenceModels.BaseDeadlineReference], ...]
```

Comment:
```suggestion
DeadlineReferenceTypes: TypeAlias = tuple[type[ReferenceModels.BaseDeadlineReference], ...]
```

not sure whether it's better this way 🤔